src/granite_orm/fields.cr

```diff
@@ -54,6 +54,30 @@ module Granite::ORM::Fields
       return parsed_params
     end
 
+    def to_h
+      fields = {} of String => Bool | Float32 | Float64 | Int32 | Int64 | String | Time | Nil
+
+      {% for name, type in FIELDS %}
+        {% if type.id == Time.id %}
+          fields["{{name}}"] = {{name.id}}.try(&.to_s("%F %X"))
+        {% elsif type.id == Slice.id %}
+          fields["{{name}}"] = {{name.id}}.try(&.to_s(""))
+        {% else %}
+          fields["{{name}}"] = {{name.id}}
+        {% end %}
+      {% end %}
+      {% if SETTINGS[:timestamps] %}
+        fields["created_at"] = created_at ? created_at.not_nil!.to_s("%F %X") : ""
+        fields["updated_at"] = updated_at ? updated_at.not_nil!.to_s("%F %X") : ""
```

Comment:
👍 

but why `""` instead of `nil` ?

```crystal
fields["created_at"] = created_at.try &.to_s("%F %X")
fields["updated_at"] = updated_at.try &.to_s("%F %X")
```

Reply:
@veelenga That makes more sense, fixed.